src/stackanalysismodule.ts

```diff
@@ -46,12 +46,11 @@ export module stackanalysismodule {
               let thatContext: any;
               options['uri'] = `${
                 Apiendpoint.STACK_API_URL
-              }stack-analyses/?user_key=${Apiendpoint.STACK_API_USER_KEY}`;
+                }stack-analyses/?user_key=${Apiendpoint.STACK_API_USER_KEY}`;
               options['formData'] = payloadData;
               options['headers'] = {
                 showTransitiveReport: 'true',
-                origin: 'vscode',
-                ecosystem: Apiendpoint.API_ECOSYSTEM
+                'x-3scale-account-secret': 'not-set'
```

Comment:
`x-3scale-account-secret` is no longer needed as we are dealing with staging data
```suggestion
```

Reply:
Are we not making these changes for production data?
For the release of free-tier?

Reply:
It was meant for devcluster, not required for staging or prod.